scripts/cli/importItems.php

```diff
@@ -0,0 +1,301 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2017 (original work) Open Assessment Technologies SA ;
+ *
+ */
+
+namespace oat\taoQtiItem\scripts\cli;
+
+use common_exception_Error;
+use common_report_Report;
+use common_report_Report as Report;
+use core_kernel_classes_Class;
+use Exception;
+use helpers_TimeOutHelper;
+use oat\generis\model\OntologyAwareTrait;
+use oat\oatbox\action\Action;
+use oat\taoQtiItem\model\qti\exception\ExtractException;
+use oat\taoQtiItem\model\qti\exception\ParsingException;
+use oat\taoQtiItem\model\qti\ImportService;
+use Zend\ServiceManager\ServiceLocatorAwareInterface;
+use Zend\ServiceManager\ServiceLocatorAwareTrait;
+
+/**
+ * Class importItems
+ *
+ * php index.php '\oat\taoQtiItem\scripts\cli\importItems' path_to_the_item_package
+ * php index.php '\oat\taoQtiItem\scripts\cli\importItems' -p path_to_the_item_package -c class_name
+ * php index.php '\oat\taoQtiItem\scripts\cli\importItems' -p path_to_the_item_package -c class_name -r -n
+ *
+ * @package oat\taoQtiItem\scripts\cli
+ * @author Jean-Sébastien Conan <jean-sebastien@taotesting.com>
+ */
+class importItems implements Action, ServiceLocatorAwareInterface
+{
+    use OntologyAwareTrait;
+    use ServiceLocatorAwareTrait;
+    
+    const ITEM_CLASS_URI = 'http://www.tao.lu/Ontologies/TAOItem.rdf#Item';
```

Comment:
we already have the taoOntology::ITEM_CLASS_URI

Reply:
I was aware of that. Fixed now